text/0000-fix-fees.md

```diff
@@ -0,0 +1,113 @@
+- Title: fix-fees
+- Authors: [John Tromp](mailto:john.tromp@gmail.com)
+- Start date: August 23, 2020
+- RFC PR: Edit if merged: [mimblewimble/grin-rfcs#0000](https://github.com/mimblewimble/grin-rfcs/pull/0000) 
+- Tracking issue: [Edit if merged with link to tracking github issue]
+
+---
+
+# Summary
+[summary]: #summary
+
+Change Grin's minimum relay fees to be weight proportional and make output creation cost about a Grin-cent.
+Use least significant bits in fee to specify desired tx priority.
+
+# Motivation
+[motivation]: #motivation
+
+The current fee requirements suffer from being somewhat arbitrary, and not miner incentive compatible.
```

Comment:
Write as if the RFC has already been merged, so it's not outdated the moment it gets accepted, i.e.:

_The previous fee requirements suffered..._

Apply this throughout the text so that it doesn't read as a proposal, but as an actual specification.